ci/infra/testrunner/tests/test_dockercaps.py

```diff
@@ -45,6 +45,16 @@
     containers:
     - name: app
       image: registry.suse.de/suse/containers/sle-server/15/containers/suse/sle15:15.1
+      command: ['/bin/sh', '-c', 'sleep 3600']
```

Comment:
@jordimassaguerpla how should I know? all I do here is adding a copy of the sle15 sp1 container as sle15 sp2 and update the references (to say 15.2 instead of 15.1). 

I don't know why this is there or what its good for.

Reply:
The diff shows you are adding the sleep command

Reply:
I'm not doing that, its just an artifact of how github visualizes the context.

maybe looking at this link makes this more obvious for you: 

https://github.com/SUSE/skuba/pull/1207/files/34d8b343de3635fdcd3d950e69741a5b37bcf82e?diff=split&w=1#diff-e94f8c03f3873659bd1f998d984dc18dL48

alternatively download the patch and open it in a text editor.

Reply:
Sorry but I have to insist. If you look at

https://github.com/SUSE/skuba/commit/89c1a8cedda97fd6d13be519d8a8766097a20f80.patch

I still see how you are adding a sleep command at ci/infra/testrunner/tests/test_dockercaps.py

Reply:
I am not doing that. I'm just adding a copy of the sle15 sp1 container as a sle15 sp2 container. thats it. 

compare what this PR does:

https://github.com/SUSE/skuba/blob/7ecebad9edcadf759e6b93a11578e7002a4d989c/ci/infra/testrunner/tests/test_dockercaps.py#L29-L38

with what the state is of before. I attach two screenshots. print them out, take a textmarker, and compare.

![screencapture-github-SUSE-skuba-blob-7ecebad9edcadf759e6b93a11578e7002a4d989c-ci-infra-testrunner-tests-test-dockercaps-py-2020-07-02-14_30_34](https://user-images.githubusercontent.com/1029152/86359141-ef937980-bc70-11ea-8f05-42a4db886c53.png)
![screencapture-github-SUSE-skuba-blob-983a0e3f560d1565774f37259be67a775652beff-ci-infra-testrunner-tests-test-dockercaps-py-2020-07-02-14_28_03](https://user-images.githubusercontent.com/1029152/86359156-f1f5d380-bc70-11ea-878f-991e6b83b1c1.png)

Reply:
Hi @dirkmueller 

I just realized what I was misreading... I didn't need a textmarker ;) . So you are correct, the diff output is a bit misleading. Then this LGTM. Sorry for holding this.